### ISSUE TITLE
Clarification of risk calculator

### ISSUE THREAD

**Issue by troutgit** [microCOVID/microCOVID#1300]
Am I right that risk is calculated annually, but based on weekly prevalence? Does the thermometer reflect annual risk? Should this be clarified? If risk goes down next month, then the annual risk does as well. It's still dangerous behavior this week, but should the thermometer reflect risk this week?

**Comment by beshaya** [comment]
Risk is accumulated based on individual activities, which are based on weekly prevalence. The risk budgets are presented as  based on a tolerance for getting COVID over the course of a year, but in the case of the calculator and thermometer the budget is actually weekly (annual risk / 50).

Here's an example:
* My annual risk budget is 10,000 microCOVID
* There are roughly 50 weeks in a year, so my weekly budget is 10,000 / 50 = 200 microCOVID
* If a certain activity this week incurs 100 microCOVID, that is half my budget for the week.

**Comment by tkpwaeub** [comment]
I'm not sure how much sense it makes to "annualize". As far as planning a long term strategy, it might make sense, but in the long run this is basically memoryless: however virtuous you may have been four weeks ago has very little bearing on the risks you take *right now*

**Comment by beshaya** [comment]
Ah, yes, prior probability vs posterior probability.

Let's say that, today, I decide that I want a 1% chance of getting COVID
over the next year. I decide to spend 200microCOVIDs a week. Six months
later, I say "hey, I haven't gotten COVID, so I actually have 400
microCOVIDs per week for the next 6 months." That's valid, but if in the
present day you know that you're going to do that, your risk now is 1.5%.

Put another way, if you have 1000 people who commit to 10,000 microCOVIDs
per year, you'd expect 10 of them to get COVID, regardless of how they
arrange the microCOVIDs over 52 weeks. If you have 1000 people who commit
to 5,000 microCOVIDs for 6 months, and then 10,000 microCOVIDs for the
second 6 months if they haven't gotten COVID yet, you'd expect (1000 *
0.005 + 1000 * (1-0.005) * 0.01) = 14.95 of them to get COVID (assuming a
microCOVID is exactly a 1-in-a-million chance of COVID).

Anway, it's valid to interpret this in other ways and it is valid to choose
risk tolerances in other ways. But we need _some_ way to contextualize the
number and the annual budget is a relatively easy way to swallow. Otherwise
people go "oh, a 2% chance of getting COVID is so low!" not realizing that
doing a 2% risk per week is a 75% chance of getting it in a year.

**Comment by tkpwaeub** [comment]
Maybe compare it counting the number of times you roll a million sided die - and give examples of how to compute the probability of eventually rolling a 19? I suggest 1-e^(-m/n) where m=number of microcovids and n=a million. (Note, m/n can exceed 1, which appears to be a source of confusion both on your website and other issues)

**Comment by beshaya** [comment]
I don't think the average person has intuition on how many million sided
die rolls they should tolerate for a given activity.

**Comment by tkpwaeub** [comment]
Right, that's why the difference should be clearly explained. Number of
drolls divided by number of sides of the diereally isn't the same as
probability
